tutorials/frontend/from_darknet.py

```diff
@@ -72,7 +72,7 @@
     DARKNET_URL = REPO_URL + 'lib/' + DARKNET_LIB + '?raw=true'
 elif sys.platform == 'darwin':
     DARKNET_LIB = 'libdarknet_mac2.0.so'
-    DARKNET_URL = REPO_URL + 'lib_osx/' + DARKNET_LIB + '?raw=true'
+    DARKNET_URL = 'https://github.com/siju-samuel/darknet/blob/master/lib_osx/' + DARKNET_LIB + '?raw=true'
```

Comment:
Can you now update it to point to REPO_URL, and test that it all works fine? Thanks

Reply:
Hi @tmoreau89 , I pulled the lasted code from TVM master, changed the `REPO_URL` back in `libdarknet_mac2.0.so`, and tested again on my local machine. This darknet tutorial runs without issues.

Before you merge and close this PR, do I need to show you the result of my local run? or do I need to do more tests? If not, I think you can merge and close this PR now.

This is my first time contributing to TVM. Thank you so much for all the help!

-Hao